earth_enterprise/src/installer/uninstall_fusion.sh

```diff
@@ -384,11 +384,11 @@ change_volume_ownership()
 
             local volume_name="test"
             local index=1
-            local max_index=$(expr "$(xmllint --xpath 'count(//VolumeDefList/volumedefs/item/localpath)' $CONFIG_VOLUME)")
-
+            local xpath_output=$(xml_file_get_xpath "$CONFIG_VOLUME" "//VolumeDefList/volumedefs/item/localpath")
+	    local max_index=`echo "$xpath_output" | grep localpath | wc -l`
```

Comment:
Minor: is this intended to be permanent? It looks like debug output.

Reply:
The original max index takes a count of the number of elements.  The best way I could think to do this, given the limitations was to take the output, filter on localpath, and count the number of lines i.e. number of localpaths

Reply:
Got it. Makes sense.